scripts/notify-docs/published-headless-atomic.js

```diff
@@ -0,0 +1,38 @@
+const {Octokit} = require('@octokit/rest');
```

Comment:
Updated the file to support atomic & headless, I didn't change the structure very much